lib/cc/analyzer/container.rb

```diff
@@ -116,7 +153,15 @@ def reap_running_container
       end
 
       def timeout
-        (ENV["CONTAINER_TIMEOUT_SECONDS"] || DEFAULT_TIMEOUT).to_i
+        ENV.fetch("CONTAINER_TIMEOUT_SECONDS", DEFAULT_TIMEOUT).to_i
+      end
+
+      def maximum_output_bytes
+        ENV.fetch("CONTAINER_MAXIMUM_OUTPUT_BYTES", DEFAULT_MAXIMUM_OUTPUT_BYTES).to_i
+      end
+
+      def counter_mutex
+        @counter_mutex ||= Mutex.new
```

Comment:
Yep, that's right. It could lead to a slightly inaccurate byte count if both streams fire at once. Moving into the initializer.